app/src/main/java/com/github/fribourgsdp/radio/LyricsGetter.kt

```diff
@@ -102,6 +127,28 @@ class LyricsGetter {
             }
         }
 
+        private fun cleanLyrics(lyrics : String) : String{
```

Comment:
I think you should document why you want to suppress the last 4 lines, even right now, it is clear why.